pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -1,16 +1,18 @@
-{ stdenv, fetchurl, system, makeWrapper, makeDesktopItem, autoPatchelfHook
+{ stdenv, fetchurl, system, makeWrapper, makeDesktopItem, autoPatchelfHook, env
```

Comment:
Where does this `env` comes from? Name is so generic that it is impossible to grep out what it is.

Reply:
The PR mentions `libsForQt5.env`, I guess that's what the param refers to.